userspace/sysdig/CMakeLists.txt

```
@@ -108,7 +107,7 @@ if(NOT WIN32)
 	include_directories(${PROJECT_BINARY_DIR}/driver/src)
 
 	target_link_libraries(sysdig
-		sinsp
+		$<IF:$<BOOL:${USE_BUNDLED_FALCOSECURITY_LIBS}>,sinsp,PkgConfig::LIBSINSP>
```

```suggestion
		"${SINSP_LIB}"
```
and define it with a simple
```
if(USE_BUNDLED_FALCOSECURITY_LIBS)
    set(SINSP_LIB sinsp)
else()
    set(SINSP_LIB PkgConfig::LIBSINSP)
endif()
```

I think that an approach like this will ensure a better readability of the cmake. WDYT?

---

Done, thanks for the review.